### ISSUE TITLE
/scripts/required.js not working on Linux Version

### ISSUE THREAD

**Issue by steeu** [Wakanda/wakanda-issues#36]
The required.js file in the scripts folder of the project root ([PROJECT_ROOT]/scripts/required.js) is not working in the Linux version 1.0.1.

Tried to set some default variables for config purposes, but the file is not even loaded on Linux. On Windows, everything works fine.

**Docs regarding required.js file:**
You can have a JavaScript file named “required.js” in a "scripts" folder at the root of your project's folder, it will be loaded for each JavaScript context generated on the server at the solution-level. For example, you can include a different JavaScript module using the include() function to load depending on the request received. If you use the setLoginListener( ) function, the "required.js" folder must be located at the root of your solution's folder.

Source: http://doc.wakanda.org/home2.en.html#/Architecture-of-Wakanda-Applications/Project.200-1022680.en.html

**Comment by hamzahik** [comment]
@stefaneugster did you try putting the file `required.js` at the solution level ?

**Comment by stefaneugster** [comment]
It ist working at solution level. I'm using it there for the custom login.

**Comment by stefaneugster** [comment]
The scripts folder on **Linux** at the project root level still needs start with an upper case letter in order for the `required.js` file to work. 

Would be better if the scripts folder could be configured in the project settings file as well.

**OK:**
```
[PROJECT_ROOT]/Scripts/required.js
```

**NOK:**
```
[PROJECT_ROOT]/scripts/required.js
```

**Comment by hamzahik** [comment]
> Would be better if the scripts folder could be configured in the project settings file as well.

@stefaneugster you're right, we are working on a similar proposition.

In future versions we will also add a new API in order to have a module handling the authentication (the same idea as the login-listener ) and the `required.js` file at the solution won't be necessary for that use case in particular.

**Comment by hamzahik** [comment]
@steeu fixed in the pre-release [v1.1.0.b](https://github.com/Wakanda/wakanda-digital-app-factory/releases/tag/v1.1.0.b)

**Comment by hamzahik** [comment]
1.1.0 released with the fix
